packages/wdio-allure-reporter/README.md

```diff
@@ -5,6 +5,121 @@ WDIO Allure Reporter
 
 ![Allure Reporter Example](/img/allure.png)
 
+### New Features
+
+- **Test Plan Support** - Execute only specific tests defined in a test plan file
+- **Enhanced SDK Integration** - Updated to use `allure-js-commons` v3.3.2 with improved performance
+- **Better Cucumber Support** - Improved handling of Cucumber scenarios and steps
+- **Enhanced Error Handling** - More robust error reporting and debugging capabilities
+
+### Test Plan Feature
+
+The Test Plan feature allows you to execute only specific tests defined in a JSON file. This is particularly useful for:
+- Running subset of tests in CI/CD pipelines
+- Parallel test execution
+- Test prioritization
+
+#### Configuration
+
+Create a test plan file (`.allure/testplan.json` or `testplan.json`):
+
+```json
+{
+  "version": "1.0",
+  "tests": [
+    {
+      "id": "test-001",
+      "selector": "test/index.test.ts#should generate testplan.json"
+    }
+  ]
+}
+```
+
+#### How Selectors Work
+
+Test plan selectors work by matching against the test's `fulltitle`, which includes the file path and test name. The format is:
+
+```
+<file-path>#<test-name>
+```
+
+Examples:
+- `test/index.test.ts#should generate testplan.json` - matches specific test in specific file
+- `Login Tests` - matches any test with "Login Tests" in the title
+- `User Registration` - matches any test with "User Registration" in the title
+
+The selector matching is flexible and supports:
+- **Exact matches** - full test title
+- **Partial matches** - substring of test title
+- **File-specific matches** - include file path for precise targeting
+
+#### Environment Variable
+
+You can also specify the test plan path via environment variable:
```

Comment:
I don't see another option to enable test plan, so we can add similar text somewhere at the beginning of the test plan section:

> The test plan support can be enabled by providing `ALLURE_TESTPLAN_PATH` environment variable – the reporter automatically loads the test plan file and run tests which match the selectors.